vitrina/structure/models.py

```diff
@@ -531,6 +531,7 @@ class MetadataVersion(models.Model):
         null=True,
         verbose_name=_("Bazė"),
     )
+    status = models.ForeignKey(Status, models.SET_NULL, verbose_name=_("Būsena"), null=True, blank=True)
```

Comment:
Maybe better to do  not allow status removal easily?
```suggestion
    status = models.ForeignKey(Status, models.PROTECT, verbose_name=_("Būsena"), null=True, blank=True)
```

Reply:
Changed